Header header 

#from bounding boxes
string Class
float64 probability
int16 id

#pose
float32 x
float32 y
float32 z
